### DISCUSSION TITLE
Change style while waybar is running

### DISCUSSION THREAD

**Discussion by JarKz** [Alexays/Waybar#3340] [Q&A] [answered]
Hi y'all!

I can't find a way that I can change style for the waybar while it's running. I'm not sure that closing and opening the waybar with another style is normal way.

**Comment by thelazt16** [comment, marked as answer]
Add `"reload_style_on_change": true,` in the config and any change you do on the `style.css` will be reflected right away. You can also use the debug mode `GTK_DEBUG=interactive waybar` and use the CSS tab from styling. You can also check what kind of property the module have from the CSS nodes (click on the show all objects in the top right and select CSS node from dropdown).

**Reply by JarKz** [reply]
Thank you so much!  This is just what I was looking for!
